### PR TITLE
Dfr 3080

### PR DESCRIPTION
**Pull request by ptrelease**
### Jira link

https://tools.hmcts.net/jira/browse/DFR-3080

### Change description

Changes that allow certain roles to complete an applicant in refuge question on the contested Form A journey.
Then certain roles can view that answer on the applicant tab.

### Testing done

* Applicant Sol can view new field on Form A create case journey
* Court Admin can view new field on Form A create case journey
* Applicant Sol can view new field applicant tab
* Court Admin can view new field on applicant tab
* Respondent Sol can't view new field on applicant tab
* Respondent Sol has no access to history, so can't see question there

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by jthmcts — APPROVED** [review]

**Review by so99y — DISMISSED** [review]

**Review by hamzahtahirhmcts — DISMISSED** [review]

**Review by jthmcts — APPROVED** [review]

File: definitions/contested/json/AuthorisationCaseField/Refuge-nonprod.json [hunk 1]
```diff
@@ -55,4 +76,4 @@
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "CRUD"
   }
-]
+]
```
Comment (so99y) [inline definitions/contested/json/AuthorisationCaseField/Refuge-nonprod.json:79]: EL

**Comment by so99y** [comment]
I am wondering there are different values of  "LiveFrom"

File: definitions/contested/json/AuthorisationCaseField/Refuge-nonprod.json (hunk 1 above)
Comment (ptrelease) [reply definitions/contested/json/AuthorisationCaseField/Refuge-nonprod.json:79]: Thanks

File: definitions/contested/json/CaseField/Refuge-nonprod.json [hunk 2]
```diff
@@ -17,4 +26,4 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   }
-]
+]
```
[the remark this answers is not in the document]
Comment (ptrelease) [reply definitions/contested/json/CaseField/Refuge-nonprod.json:29]: Thanks

**Review by jthmcts — APPROVED** [review]

**Review by so99y — APPROVED** [review]

**Pull request merged**